src/main/java/org/openlmis/buq/service/buq/BottomUpQuantificationService.java

```diff
@@ -92,6 +95,9 @@ public class BottomUpQuantificationService {
   @Autowired
   private BottomUpQuantificationRepository bottomUpQuantificationRepository;
 
+  @Autowired
+  private RemarkRepository remarkRepository;
+
```

Comment:
I think you should've injected the `RemarkService`

Reply:
I changed repository usage to service